NEWS.md

```diff
@@ -84,6 +84,12 @@
 
 * Increase the default `nbin` of `guide_colourbar()` to place the ticks more precisely (#3508, @yutannihilation).
 
+* Modified `manual_scale` to match `values` with the order of `breaks` whenever
```

Comment:
Please write "`manual_scale()` now matches `values` with ..." Note the parentheses at the end of `manual_scale()`.